api/options.ts

```diff
@@ -0,0 +1,28 @@
+import axios from "axios";
+import { IUser } from "types/User";
+const baseURL = "http://localhost:8080/api";
+const headers = {
+  "Content-Type": "application/json",
+  Authorization: "Bearer ",
+};
+
+const OptionsAPI = {
+  create: async (user: IUser) => {
+    return await axios.post(`${baseURL}/cresources`, user, { headers });
+  },
+  getById: async (_id: number) => {
+    return await axios.get(`${baseURL}/cresources/${_id}`, { headers });
+  },
+  getAll: async () => {
+    return await axios.get(`${baseURL}/cresources`, { headers });
+  },
+  delete: async (_id: number) => {
+    return await axios.delete(`${baseURL}/cresources/${_id}`, { headers });
+  },
+  update: async (user: IUser) => {
+    return await axios.put(`${baseURL}/cresources/${user._id}`, user, {
+      headers,
+    });
+  },
+};
+export default OptionsAPI;
```

Comment:
Ich glabe options brauchen wir nicht mehr? Lass uns jetzt bei users nur noch bleiben

Reply:
Wie ich mir das dachte: Options ist die übliche Seite auf der ein user seine eigenen Daten verändern kann/den Account löscht, sollte man ja drin haben. In der User Seite kann zum einen ausschließlich der Admin (muss dann später für normale user ausgeblendet werden) alle user bearbeiten. Zum andern sollte die Seite dann Ausgangspunkt für diese Gruppen von usern die gemeinsam Cluster reservieren wollen werden.